src/multi-http-loader.ts

```diff
@@ -13,14 +13,14 @@ export interface ITranslationResource {
 
 export class MultiTranslateHttpLoader implements TranslateLoader {
   constructor(
-    private http: HttpClient,
+    private handler: HttpBackend,
     private resources: ITranslationResource[],
   ) {}
 
   public getTranslation(lang: string): Observable<any> {
     const requests = this.resources.map(resource => {
       const path = resource.prefix + lang + resource.suffix;
-      return this.http.get(path).pipe(catchError(res => {
+      return new HttpClient(this._handler).get(path).pipe(catchError(res => {
```

Comment:
Does this work? The field is named `handler` in the constructor, without the underscore _ .

Reply:
You're right! I removed quickly the `_` in the constructor cause the way is develop do not use it. Forget to remove it also down there, sorry